### ISSUE TITLE
Some punctuation marks are misinterpreted by circuits due to moldy spaghetti

### ISSUE THREAD

**Issue by QuarterEighth** [CSDSP/ss13-circuit-visualizer#1]
Title. SS13 is an old game built on an older engine (BYOND), and over the years things have happened such that the circuit printer gets notably confused when you feed it certain grammatical symbols, usually a Quotation Mark ( `"` unicode 0x22) or an apostrophe ( `'` unicode 0x27). How often you're going to do this is questionable, but I figure it's worth pointing out that in at least these two cases, you can avoid the problem by switching them (for TTS circuit use) for `’` and `”` (being 0x2019 and 0x201d), **or** (for making it so the printer can read it and not get tripped up) just replacing them with their unicode values.

In summary, if you could add a bit of code that changes apostrophes and quotation marks (when placed into "string" fields) into unicode addresses (eg. \u0022 \u0027) when compiling into code, that would be great. Reply or PM for more info if needed.
